t/proto_template/testdata/golden/basic/data/scripts/generate.sh

```diff
@@ -26,6 +26,11 @@ function post_gen_go {
 function pre_gen_node {
   echo "running pre_gen_node"
 
+  proto_version_key='proto_version:'
+  proto_version_file='version.yaml'
+
+  PROTO_VERSION=$(grep $proto_version_key $proto_version_file | cut -c $((${#proto_version_key} + 1))- | xargs)
```

Comment:
I changed this a bit to check if the variable is already set - https://github.com/TapestryEnergy/tapestry-shared-protos/pull/1/files#diff-25806c0c4c68cee274f0c43c1b0ed2c9579bea4c3816903bfad038249cf7e934R33